python/lsst/rubintv/production/rubinTv.py

```diff
@@ -825,3 +850,201 @@ def beat(self, ensure=False, customFlatlinePeriod=None):
+                                           config=self.calibrate.config.astromRefObjLoader)
+            self.calibrate.astrometry.setRefObjLoader(loader)
+            loader = self._getRefObjLoader(self.calibrate.config.connections.photoRefCat, visitDataId,
+                                           config=self.calibrate.config.photoRefObjLoader)
+            self.calibrate.photoCal.match.setRefObjLoader(loader)
+
+            charRes = self.charImage.run(exp)
+            tCharacterize = time.time()
+            self.log.info(f"Ran characterizeImageTask in {tCharacterize-tStart:.2f} seconds")
+
+            nSources = len(charRes.sourceCat)
+            dayObs = butlerUtils.getDayObs(dataId)
+            seqNum = butlerUtils.getSeqNum(dataId)
+            outputDict = {"50-sigma source count": nSources}
+            mdDict = {seqNum: outputDict}
+            writeMetadataShard(self.shardsDir, dayObs, mdDict)
+
+            calibrateRes = self.calibrate.run(charRes.exposure,
+                                              background=charRes.background,
+                                              icSourceCat=charRes.sourceCat)
+            tCalibrate = time.time()
+            self.log.info(f"Ran calibrateTask in {tCalibrate-tCharacterize:.2f} seconds")
+
+            summaryStats = calibrateRes.outputExposure.getInfo().getSummaryStats()
+            pixToArcseconds = calibrateRes.outputExposure.getWcs().getPixelScale().asArcseconds()
+            SIGMA2FWHM = np.sqrt(8 * np.log(2))
+            e1 = (summaryStats.psfIxx - summaryStats.psfIyy) / (summaryStats.psfIxx + summaryStats.psfIyy)
+            e2 = 2*summaryStats.psfIxy / (summaryStats.psfIxx + summaryStats.psfIyy)
+
+            outputDict = {
+                '5-sigma source count': len(calibrateRes.outputCat),
+                'PSF FWHM': summaryStats.psfSigma * SIGMA2FWHM * pixToArcseconds,
+                'PSF e1': e1,
+                'PSF e2': e2,
+                'Sky mean': summaryStats.skyBg,
+                'Sky RMS': summaryStats.skyNoise,
+                'Variance plane mean': summaryStats.meanVar,
+                'PSF star count': summaryStats.nPsfStar,
+                'Astrometric bias': summaryStats.astromOffsetMean,
+                'Astrometric scatter': summaryStats.astromOffsetStd,
+                'Zeropoint': summaryStats.zeroPoint
+            }
+
+            # flag all these as measured items to color the cell
+            labels = {"_" + k: "measured" for k in outputDict.keys()}
+            outputDict.update(labels)
+
+            mdDict = {seqNum: outputDict}
+            writeMetadataShard(self.shardsDir, dayObs, mdDict)
+            self.log.info(f'Wrote metadata shard. Putting calexp. {dataId}')
+            self.safeWrite(calibrateRes.outputExposure, "calexp", dataId)
+            tFinal = time.time()
+            self.log.info(f"Ran everything in {tFinal-tStart:.2f} seconds")
```

Comment:
Maybe change "everything" to either characterize + calibrate, or "single frame processing" or something?